Tests/OptimizelyTests-Common/BatchEventBuilderTests_Events.swift

```diff
@@ -28,6 +28,8 @@ class BatchEventBuilderTests_Events: XCTestCase {
     let datafile = OTUtils.loadJSONDatafile("api_datafile")!
     
     override func setUp() {
+        Utils.sdkVersion = OPTIMIZELYSDKVERSION
+        Utils.swiftSdkClientName = "swift-sdk"
```

Comment:
Do we need this lines for testing? It may be more interesting without them to check if all works fine with default config.

Reply:
Multiple tests started failing since these are global variables and the changes were reflected on all other unit tests. Some of our unit tests assert for swiftSdkClientName to be "swift-sdk".